README.md

```diff
@@ -35,28 +34,27 @@ Run `optimake .` in the folder containing `structures.zip` and `optimade.yaml` t
 
 See `./examples` for other supported formats and corresponding `optimade.yaml` config files.
 
-### optimake JSONLines Format
+## optimake JSONLines Format
 
-As described above, `optimake` works via an intermediate JSONLines file representation of an OPTIMADE API.
+As described above, `optimake` works via an intermediate JSONLines file representation of an OPTIMADE API (see also the [corresponding issue in the specification](https://github.com/Materials-Consortia/OPTIMADE/issues/471)).
 This file should provide enough metadata to spin up an OPTIMADE API with many different entry types.
 The format is as follows:
 
-- First line must be a dictionary with the key x-optimade-api-version with value e.g., 1.2.0.
-- Then comes any other metadata under meta
-- Then corresponding info endpoint data for the types in the file
-- Then each line contains an entry from the corresponding endpoints
+- First line must be a dictionary with the key `x-optimade`, containing a sub-dictionary of metadata (such as the OPTIMADE API version).
+- Second line contains the `info/structures` endpoint.
+- Third line contains the `info/references` endpoint, if present.
+- Then each line contains an entry from the corresponding individual structure/reference endpoints.
 
 ```json
-{"x-optimade": {...}}
-{"meta": {"api_version": "1.1.0"}}
-{"data": {"type": "info", "attributes": {"provider": {...}}}
-{"info/structures": {"properties": ["_aiida_cell_volume": {"type": "number", ...}]}
-{"info/references": {...}}
-{"data": {"type": "structures", "id": "1234", "attributes": {...}}
-{"data": {"type": "structures", "id": "1235", "attributes": {...}}
-{"data": {"type": "references", "id": "sfdas", "attributes": {...}}
+{"x-optimade": {"meta": {"api_version": "1.1.0"}}}
+{"properties": {...}}
+{"properties": {...}}
```

Comment:
```suggestion
{"type": "info", "id": "structures", "properties": {...}}
{"type": "info", "id": "references", properties": {...}}
```

minor suggestions that better document the format wrt. OPTIMADE v1.2. I think it is better to add them here (as well as your sentence below)